tests/console/validate_lvm_raid1.pm

```diff
@@ -25,20 +25,18 @@ sub run {
 
     select_console 'root-console';
 
-    my $config = get_test_suite_data();
-    $config->{expected_num_devs} = scalar @{$config->{disks}};
-    # actual_num_devs is used to get the number of the disks in raid
+    my $config            = get_test_suite_data();
+    my $expected_num_devs = scalar @{$config->{disks}};
+    # expected_num_devs is used to get the number of the disks in raid
     # as we move in and out a disk during the test. This is because
     # after the reboot sle15 can recover the missing disk and we need
     # to compare with the expected_num_devs
```

Comment:
i think you can remove the comment completely. or you have to rephrase it. Especially the last line. But honestly i would remove it. i thought it would make sense to put it as i had create another variable ("actual_num_devs"). However you have simplified this part and i dont think the comment is necessary any more.

Reply:
Indeed. Looks pretty clear what "scalar @{$config->{disks}}" will return.